pkg/externalresource/external_resource.go

```diff
@@ -10,14 +10,18 @@ import (
 	"github.com/spf13/afero"
 )
 
+const (
+	urlEnvFormat = "OKTETO_EXTERNAL_%s_ENDPOINTS_%s_URL"
+)
+
 // ExternalResourceSection represents the map of external resources at a manifest
 type ExternalResourceSection map[string]*ExternalResource
 
 // ExternalResource represents information on an external resource
 type ExternalResource struct {
 	Icon      string
 	Notes     *Notes
-	Endpoints []ExternalEndpoint
+	Endpoints []*ExternalEndpoint
```

Comment:
Why do we need to convert it to a pointer?

Reply:
I did it to update the object by passing the struct of the endpoints by reference